selfservice/hook/session_destroyer_test.go

```diff
@@ -0,0 +1,46 @@
+package hook_test
+
+import (
+	"context"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/ory/viper"
+
+	"github.com/ory/kratos/driver/configuration"
+	"github.com/ory/kratos/identity"
+	"github.com/ory/kratos/internal"
+	"github.com/ory/kratos/selfservice/hook"
+	"github.com/ory/kratos/session"
+	"github.com/ory/kratos/x"
+)
+
+func TestSessionDestroyer(t *testing.T) {
+	_, reg := internal.NewRegistryDefault(t)
+	viper.Set(configuration.ViperKeyURLsSelfPublic, "http://localhost/")
+	viper.Set(configuration.ViperKeyDefaultIdentityTraitsSchemaURL, "file://./stub/stub.schema.json")
+
+	var r http.Request
+	h := hook.NewSessionIssuer(reg)
+
+	t.Run("method=ExecuteLoginPostHook", func(t *testing.T) {
+		w := httptest.NewRecorder()
+		sid := x.NewUUID()
+
+		i := identity.NewIdentity("")
+		require.NoError(t, reg.IdentityPool().CreateIdentity(context.Background(), i))
+		err := reg.SessionPersister().CreateSession(context.Background(), &session.Session{ID: sid, Identity: i}))
+		require.NoError(t, err)
+		err = reg.SessionPersister().CreateSession(context.Background(), &session.Session{ID: sid, Identity: i}))
```

Comment:
This creates the same session twice, instead, it would be better to create two individual sessions with individual IDs

Reply:
are you saying that i need two sid ?
```
err := reg.SessionPersister().CreateSession(context.Background(), &session.Session{ID: sid, Identity: i}))
err := reg.SessionPersister().CreateSession(context.Background(), &session.Session{ID: sid2, Identity: i}))
```

Reply:
done